projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -103,14 +112,13 @@ export class GoTableComponent implements OnInit, OnChanges {
   }
 
   toggleSort(columnField: string): void {
-    const { sortable, sortConfig, tableData }:
+    const { sortConfig, tableData }:
     {
-      sortable: boolean,
       sortConfig?: GoTableSortConfig,
       tableData: any[]
     } = this.localTableConfig;
 
-    if (tableData && sortable) {
+    if (tableData) {
```

Comment:
I think we want to keep this because this will restrict sorting on a table level, not a column level.